public/package.json

```diff
@@ -5,8 +5,8 @@
   "homepage": "",
   "author": "Bitovi",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "install": "node build.js"
+    "install": "node build.js",
+    "test": "testee test.html --browsers firefox --reporter Spec"
```

Comment:
The example Justin linked to used FF. I think testing in a "real" browser is something we want